slimta/edge/wsgi.py

```diff
@@ -158,17 +194,18 @@ def __init__(self, queue, hostname=None, validator_class=None,
                  uri_pattern=None):
         super(WsgiEdge, self).__init__(queue, hostname)
         self.validator_class = validator_class
-        if isinstance(uri_pattern, basestring):
+        if isinstance(uri_pattern, six.string_types):
             self.uri_pattern = re.compile(uri_pattern)
         else:
             self.uri_pattern = uri_pattern
 
     def __call__(self, environ, start_response):
-        self._trigger_ptr_lookup(environ)
+        unicode_environ = decode_environ(environ)
```

Comment:
Change arg names would possibly break *WsgiServer* API, leading to possible errors now or later (if some code is using keyword-arguments). So I'm not in favor.